testutil/compose/testdata/TestLockCompose.golden

```diff
@@ -0,0 +1,22 @@
+version: "3.8"
+
+x-node-base: &node-base
+  image: ghcr.io/obolnetwork/charon:latest
+  entrypoint: /usr/local/bin/charon
+  command: [create,cluster]
```

Comment:
is this means space separated command names?
like charon create dkg would be `[create, dkg]`?

Reply:
yeah, otherwise it is a single argument `charon create cluster` vs `charon 'create cluster'`